stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -57,6 +57,11 @@ public func enableGPU() {
   #tfop("tfc.configureGPU") as Void
 }
 
+@_transparent
+public func enableCPU() {
```

Comment:
This name sounds really confusing to me: CPU is always enabled. Do you want to use something like `useCPU`?

Reply:
Richard, what do you think about an API like:
useDevice(.CPU)

?

If you agree that this is the right direction, could you tackle this as a follow-up patch?

Reply:
Agree the name is confusing. I'll leave it to Richard to design a better swift level API. :) Filed https://bugs.swift.org/browse/SR-8517 for the backend side of work.

Reply:
SG. Thanks.